tracer/src/Datadog.Trace/Logging/TracerFlare/TracerFlareRequestFactory.cs

```diff
@@ -0,0 +1,82 @@
+﻿// <copyright file="TracerFlareRequestFactory.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Datadog.Trace.Util;
+
+namespace Datadog.Trace.Logging.TracerFlare;
+
+internal class TracerFlareRequestFactory
+{
+    internal const string Boundary = "--83CAD6AA-8A24-462C-8B3D-FF9CC683B51B";
```

Comment:
We could use utf-8 strings to have the compiler do the encoding work, e.g. 
```suggestion
    private static byte[] Boundary = "--83CAD6AA-8A24-462C-8B3D-FF9CC683B51B"u8.ToArray();
```

I didn't bother because
- We can't use the `ReadOnlySpan<byte>` trick because they're used in async code, which means we need to create an array
- We can't use interpolation in the raw string literals, so it's more chunks to worry about
- We can't use this in .NET FX at all, which means two implementations (yuk)

Reply:
I forgot about the `u8` stuff in C#11, that would be nice but I'd agree that having to have split implementations between .NET Framework/Standard and .NET Core.

> We can't use interpolation in the raw string literals, so it's more chunks to worry about

I didn't know that